ynr/apps/people/forms/forms.py

```diff
@@ -364,16 +364,11 @@ class Meta:
     )
 
     def clean_biography(self):
-        if self.cleaned_data["biography"].find("\r"):
-            self.cleaned_data["biography"] = self.cleaned_data[
-                "biography"
-            ].replace("\r", "")
-        return "\n\n".join(
-            [
-                line.replace("\n", " ")
-                for line in self.cleaned_data["biography"].split("\n\n")
-            ]
-        )
+        bio = self.cleaned_data["biography"]
+        if bio.find("\r"):
+            bio = bio.replace("\r", "")
+        # Reduce > 2 newlines to 2 newlines
+        return "\n\n".join([line for line in bio.split("\n\n") if line.strip()])
```

Comment:
This won't quite do what we want if the number of `\n`s is odd (rather than even).

```suggestion
        return "\n\n".join([line.strip() for line in bio.split("\n\n") if line.strip()])
```

should fix that though

Reply:
Ah right. Done in d482646fcb790bbcfa2c1855eab84a95ba36be7b